### PR TITLE
Finish front end

### PR DESCRIPTION
**Pull request by stbensonimoh**

### CODE REVIEW

File: src/pages/apply/FormPage.js [hunk 1]
```diff
@@ -31,274 +32,554 @@ const FormPage = () => {
...
+              linkedinHandle: "",
+              publicFigures: "",
+              mostAdmiredNGO: "",
+              preferredSocialMedia: "",
+              volunteerType: "",
+              interestedVolunteerUnits: [],
+              reasonForVolunteering: "",
+            }}
+            onSubmit={(values) => {
+              const formattedData = {
```
Comment (osumgbachiamaka) [inline src/pages/apply/FormPage.js:57]: We can assign a separate function to handle this object construction

So onSubmit does its work of submitting

File: src/pages/apply/FormPage.js [hunk 2]
```diff
@@ -31,274 +32,554 @@ const FormPage = () => {
...
+                }
+              )
+                .then((response) => {
+                  if (response.status === 201) {
+                    Swal.fire({
+                      title: "Application Successful!",
+                      text: "Check your email for the next steps.",
+                      icon: "success",
+                    });
+                    setTimeout(() => {
```
Comment (osumgbachiamaka) [inline src/pages/apply/FormPage.js:100]: setTimeout can be removed from the if blocks and called once immediately after the if blocks.

The if blocks are merely validating the status codes.
We can redirect afterwards

File: src/pages/apply/FormPage.js [hunk 3]
```diff
@@ -31,274 +32,554 @@ const FormPage = () => {
...
+                      title: "Already Registered",
+                      text: "You have already registered",
+                      icon: "warning",
+                    });
+                    setTimeout(() => {
+                      window.location = "https://nhfafrica.org";
+                    }, 3000);
+                  }
+                })
+                .then((responseData) => {
```
Comment (osumgbachiamaka) [inline src/pages/apply/FormPage.js]: responseData here is the same as in the first .then()

**Review by osumgbachiamaka — APPROVED** [review]
Aside from the comments mentioned, the work looks good.

File: src/pages/apply/FormPage.js (hunk 3 above)
Comment (stbensonimoh) [reply src/pages/apply/FormPage.js]: Thanks! This has been resolved.

File: src/pages/apply/FormPage.js (hunk 1 above)
Comment (stbensonimoh) [reply src/pages/apply/FormPage.js:57]: This will be resolved in the next life 🤗

File: src/pages/apply/FormPage.js (hunk 2 above)
Comment (stbensonimoh) [reply src/pages/apply/FormPage.js:100]: and this too 🤕

**Pull request merged**